app/controllers/user.js

```diff
@@ -1,16 +1,41 @@
 const validations = require('../util');
```

Comment:
Rename for `utils`. Remember, you must be declarative. Also, the file must be `utils.js`